app/views/auth/omniauth_registrations/new.html.slim

```diff
@@ -0,0 +1,14 @@
+section.row-0
+  .center-panel
+    .col-md-4.col-sm-2.col-xs-1
+    .col-md-4.col-sm-8.col-xs-10.text-center
+      = render 'shared/notifications'
+      = image_tag 'layout/portus-logo-login-page.png', class: 'login-picture'
+      .row
+        .col-sm-12
+          h5.create-new-account Define your Username and Display Name
```

Comment:
I would suggest to change to change this to `p` and ignore the `.col-sm-12` and `.row`, going right below the `image_tag`. 

You can also associate it with a class and style it as you might need.